0.10/s2i/bin/assemble

```diff
@@ -47,8 +47,20 @@ if [ -n "$NPM_MIRROR" ]; then
 	npm config set registry $NPM_MIRROR
 fi
 
+if [ ! -z $NPM_LOG_LEVEL && $NPM_LOG_LEVEL == "info" ]; then
+	echo "--> ${NPM_LOG_LEVEL} set. Setting npm logging level to 'info'"
+	npm config set loglevel info
+
+elif [ ! -z $NPM_LOG_LEVEL && $NPM_LOG_LEVEL == "silent" ]; then
+	echo "--> ${NPM_LOG_LEVEL} set. Setting npm logging level to 'silent'"
+	npm config set loglevel silent
+else
+	echo "--> No NPM_LOG_LEVEL environment variable set. Setting npm logging level to default 'warn'"
+	npm config set loglevel warn
+fi
```

Comment:
@bparees I think this makes sense actually because we can just assign the default value to warn if its not set